UC-Berkeley-Extension/global
# Panorama / Testimonial components

## Asana Ticket(s)

## Description
- Config:
  - Creates Landing Page content type
  - Enables Media
  - Creates Panorama and Testimonial paragraphs, and adds them as entity reference to the Landing Page Components paragraph.
- Styleguide markup for Panorama and Testimonial components

## Testing instructions
- Rebuild the style guide and theme dependencies:
 - From the host machine:
 - `cd web/themes/custom/bglobal`
 - `rm -rf node_modules`
 - `npm install`
- Serve the styleguide (within web/themes/custom/bglobal dir, run `npm start`)
- Visit molecules -> Panorama and Testimonial components, make sure they are there.
  - note: Testimonial component is not styled.
- Within the container (after `ddev ssh`):
  - `drush cim`, `drush cr`
  - Create a Landing Page (https://global.ddev.site/node/add/landing_page), and add a Panorama component and a Testimonial component.
  - Confirm that the components are rendering as expected, same as the styleguide.
